pkg/executor/join.go

```diff
@@ -1633,7 +1661,7 @@ func (e *hashJoinRuntimeStats) String() string {
 		buf.WriteString(execdetails.FormatDuration(time.Duration(atomic.LoadInt64(&e.maxFetchAndProbe))))
 		buf.WriteString(", probe:")
 		buf.WriteString(execdetails.FormatDuration(time.Duration(e.probe)))
-		buf.WriteString(", fetch:")
+		buf.WriteString(", fetch and wait:")
```

Comment:
Add a comment for what does `fetch and wait` mean?

Reply:
Done